### PR TITLE
Config info for GitLab Client-Side Implicit Grant

### PR DESCRIPTION
**Pull request by jbialobr**
## Description

As per comment https://github.com/netlify/netlify-cms/issues/1690#issuecomment-465078677
Identity widget conflicts with implicit authentication and should be disabled.

### CODE REVIEW

File: docs/docs/sourcing-from-netlify-cms.md [hunk 1]
```diff
@@ -144,6 +144,23 @@ backend:
 
 Now you can save the config.yml file, commit the change, and push it to your GitHub repo.
 
+#### Authenticating with GitLab
+
+See the [GitLab Backend](https://www.netlifycms.org/docs/authentication-backends/#gitlab-backend)
+section for details on how to configure authentication with GitLab.
+
+If you use the Client-Side Implicit Grant option, disable the Netlify Identity service in your
```
Comment (LekoArts) [inline docs/docs/sourcing-from-netlify-cms.md]:
```suggestion
If you use the [Client-Side Implicit Grant](https://www.netlifycms.org/docs/authentication-backends/#client-side-implicit-grant) option, disable the Netlify Identity service in your
```

How about also linking this? 😊

Comment (jbialobr) [reply docs/docs/sourcing-from-netlify-cms.md]: Sure, good idea. Should I amend my commit or just commit your suggestion and let the merge/squash decision to the maintainer?

Comment (LekoArts) [reply docs/docs/sourcing-from-netlify-cms.md]: You can click on "Commit suggestion" :)

image

The PR gets squashed to one commit

Comment (jbialobr) [reply docs/docs/sourcing-from-netlify-cms.md]: Thank you.

**Pull request merged**